dipy/segment/clustering.py

```diff
@@ -2,6 +2,7 @@
 import numpy as np
```

Comment:
I noticed that the coverage in clustering.py is at 86% is that the best it can be?